configure.ac

```diff
@@ -1721,7 +1721,7 @@ if test x$need_bundled_univalue = xyes; then
   AC_CONFIG_SUBDIRS([src/univalue])
 fi
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --with-bignum=no --enable-module-recovery --disable-jni"
+ac_configure_args="${ac_configure_args} --enable-option-checking --disable-shared --with-pic --enable-benchmark=no --with-bignum=no --enable-module-recovery"
```

Comment:
This isn't really ergonomic, because `ac_configure_args="${ac_configure_args}..."` passes to secp configure everything you pass into bitcoin's configure (not sure if this is wanted behavior or not), but now with `--enable-option-checking` if I pass any bitcoin specific flags like `./configure --with-incompatible-bdb --with-libs --with-daemon --with-gui` I get a warning:
`configure: WARNING: unrecognized options: --with-incompatible-bdb, --with-libs, --with-daemon, --with-gui`
`configure: WARNING: unrecognized options: --with-incompatible-bdb, --with-libs, --with-daemon, --with-gui`

Reply:
Right. After suggesting I started working on a change to fix this up the problem you mention (configure options getting passed around), as it's not very clean at the moment. We can just revert the addition of `--enable-option-checking` for now and drop `--disable-jni`.